File: Programmers/Level2/최댓값과 최솟값.js
//2021.11.13
function solution(s) {
  let arr = s.split(" ");
  for (let n = 0; n < arr.length; n++) {
    arr[n] = parseInt(arr[n], 10);
  }

  return String(Math.min(...arr) + " " + Math.max(...arr));
}
// 군더더기 없음

File: Programmers/Level2/최댓값과 최솟값.py
# 2023.04.13
def solution(s):
    string_list = s.split(' ')
    max_v = string_list[0]
    min_v = string_list[0]

    for target in string_list:
        if int(target) > int(max_v):
            max_v = target
        if int(target) < int(min_v):
            min_v = target

    return min_v + ' ' + max_v